micro-rdk-server/build.rs

```diff
@@ -36,16 +36,14 @@ fn main() {
         embuild::build::CfgArgs::output_propagated("MICRO_RDK").unwrap();
         embuild::build::LinkArgs::output_propagated("MICRO_RDK").unwrap();
     }
-    if std::env::var_os("MICRO_RDK_WIFI_PASSWORD").is_some() {
-        if let Ok(content) = std::fs::read_to_string("viam.json") {
-            if let Ok(cfg) = serde_json::from_str::<Config>(content.as_str()) {
-                println!(
-                    "cargo:rustc-env=MICRO_RDK_ROBOT_SECRET={}",
-                    cfg.cloud.secret
-                );
-                println!("cargo:rustc-env=MICRO_RDK_ROBOT_ID={}", cfg.cloud.id);
-                println!("cargo:rustc-cfg=has_robot_config");
-            }
+    if let Ok(content) = std::fs::read_to_string("viam.json") {
```

Comment:
I don't think it's necessary, since it will have no effect; it's not referenced in `native/main.rs`. 
It's also one less thing to keep track of when hopping between native and esp32 builds.